monitoring/alert-rules.yml

```
@@ -264,3 +264,43 @@ groups:
             The last successful secret rotation was {{ $value | humanizeTimestamp }}.
             Rotations are scheduled quarterly (every 91 days). Consider running
             the workflow manually if the automated schedule has been failing.
+
+
+      - name: stellar-indigopay-ai-summary
+    interval: 30s
```

YAMLlint (1.37.1)

[error] duplication of key "interval" in mapping

(key-duplicates)